optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml

```diff
@@ -1078,8 +1082,32 @@
   &lt;/solverBenchmark&gt;
 &lt;/#list&gt;
 &lt;/#list&gt;
+
+&lt;#list [1, 100, 1000] as acceptedCountLimit&gt;
+&lt;#list [1hard/5medium/10soft, 1hard/10medium/20soft, 1hard/20medium/50soft, 1hard/50medium/70soft] as startingTemperature&gt;
+  &lt;solverBenchmark&gt;
+    &lt;name&gt;SimulatedAnnealing acceptedCountLimit ${acceptedCountLimit} temperature ${startingTemperature?replace("/", "_")}&lt;/name&gt;
+    &lt;solver&gt;
+      &lt;localSearch&gt;
+        &lt;acceptor&gt;
+          &lt;simulatedAnnealingStartingTemperature&gt;${startingTemperature?replace("/", "_")}&lt;/simulatedAnnealingStartingTemperature&gt;
```

Comment:
in this element there shouldn't be a replace.

Did you run this code? I 'll try to run it and fix it.